src/main/java/frc/robot/RobotContainer.java

```diff
@@ -8,45 +8,25 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
-import frc.robot.subsystems.Drivetrain;
-import monologue.Logged;
-import static frc.robot.Constants.ControllerConstants.*;
-
-public class RobotContainer implements Logged {
-  CommandXboxController m_driverController = new CommandXboxController(DriverXbox.kControllerID);
-  Drivetrain m_drivetrain = new Drivetrain();
-
 import edu.wpi.first.wpilibj2.command.*;
-import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
 import frc.robot.subsystems.*;
-
 // import frc.robot.subsystems.SpinNEOS;
 // import monologue.Logged;
 import frc.robot.subsystems.vision.Optometrist;
+import monologue.Logged;
+import static frc.robot.Constants.ControllerConstants.*;
 
-
-public class RobotContainer {
-
-  CommandXboxController m_driverController = new CommandXboxController(0);
+public class RobotContainer implements Logged {
+  CommandXboxController m_driverController = new CommandXboxController(DriverXbox.kControllerID);
```

Comment:
IMO having this many nested classes in Constants is overkill. I could see nesting things this much if we end up making the controls code beefier, but **until** we do, I don't see why we can't just have a `ControllerConstants` subclass by itself. 

Besides, we haven't even determined whether we'll even **need** an operator. I'm going to be much more adamant about needing to see strong arguments in favor of adding it in this time around -- IMO it just makes things more complicated unnecessarily more often than not.